### ISSUE TITLE
Lint/CopDirectiveSyntax gives false positives for inline cops

### ISSUE THREAD

**Issue by syakovyn** [rubocop/rubocop#13959]
Given account.rb:
```ruby
# frozen_string_literal: true

class Account # rubocop:enable Metrics:AbcSize
  TEST = 'test'
end
```

and .rubocop.yml:
```yaml
AllCops:
  NewCops: enable
```

when running `rubocop account.rb`

I'm getting:

```
Inspecting 1 file
W

Offenses:

account.rb:3:15: W: Lint/CopDirectiveSyntax: Malformed directive comment detected. Cop names must be separated by commas. Comment in the directive must start with --.
class Account # rubocop:enable Metrics:AbcSize
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

The issue happens for any Metrics cop, but it doesn't happen for other categories, e.g. Layout or Lint

## Expected behavior

No warning is given for the case in the ticket

## Actual behavior

The warning is produced

## Steps to reproduce the problem

Create account.rb and .rubocop.yml. Run `rubocop account.rb`

## RuboCop version

```
rubocop -V
1.73.2 (using Parser 3.3.7.1, rubocop-ast 1.38.1, analyzing as Ruby 3.1, running on ruby 3.1.4) +server [arm64-darwin23]
```

**Comment by Earlopain** [comment]
Hi! This is actually correct.

You are disabling the entire `Metrics` department because `:` is the delimiter, which it terminates the cop list. You should instead write `Metrics/AbcSize`.

It's not the first time I see this, the message can probably include that `/` should be used.

**Comment by syakovyn** [comment]
I'm so sorry. I didn't notice they differ (separated by colon `:` instead of `/`). Thank you, @Earlopain, for pointing that out!
